### ISSUE TITLE
show options from own TCA when custom fields are reused

### ISSUE THREAD

**Issue by medarob** [Gernott/mask#581]
When a custom field was choosen to be reused in mask, the label, descriptions, options and probably other values are not shown in the mask modul backend.

It looks "empty" like this:
![grafik](https://github.com/Gernott/mask/assets/33951787/2aa5ccc1-383d-4ca5-a829-0d7c1b1966ad)

It would be great if the current label and description text from the own custom TCA settings could be shown as placeholder text.
Maybe with a checkbox before the field and if checked, it's possible to overwrite it (like you can overwrite image title and description in an Textmedia element).
In case of a select field it would be nice to show the current options for the Elemente item as well. (but not editable?)

Maybe other options can be shown as well if available?

**Question:**
It seems that I can overwrite every option, for example the Elemente items?
Maybe it's possible to change individual options or add additional Elemente items?

**Comment by nhovratov** [comment]
> It seems that I can overwrite every option, for example the Elemente items?
Yes, since Mask 8.2 this is possible. You can override almost every option from "real" TCA defined fields.

> Maybe it's possible to change individual options or add additional Elemente items?
I think overriding items does not really work, as these are indexed array items and can't be easily replaced. If you add e.g. an item, the first one will be replaced, because they share the same key "0".

**Comment by nhovratov** [comment]
Mask won't receive any more features.
